### ISSUE TITLE
What if I didn't use BL30K?

### ISSUE THREAD

**Issue by longmalongma** [hkchengrex/Mask-Propagation#14]
I did not use your pretrainined model and started training your network again. My pre-training only used static pictures instead of BL30K. After the completion of the training, the result of the test on Davis 2017 data set was only 70.6%(J F mean), is this result normal?What would the normal result be if we didn't use BL30K?

![image](https://user-images.githubusercontent.com/40291722/115486257-e1a6f780-a288-11eb-97ea-590c282c88aa.png)

**Comment by hkchengrex** [comment]
You should get a J&F of `83.1`. This is provided in a table of the readme.

**Comment by longmalongma** [comment]
> You should get a J&F of `83.1`. This is provided in a table of the readme.

OK, I will retrain again.
